src/model/snippet.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Tuple, Optional, List
+
+from src.model.license_expression import LicenseExpression
+from src.model.spdx_no_assert import SpdxNoAssert
+from src.model.spdx_none import SpdxNone
+
+
+class Snippet:
+    # Mandatory properties
+
+    spdx_id: str
+    file_spdx_id: str
+    byte_range: Tuple[int, int]
+
+    # Optional properties
+
+    line_range: Optional[Tuple[int, int]]
+    concluded_license: Optional[LicenseExpression, SpdxNoAssert, SpdxNone]
+    license_info_in_snippet: Optional[List[LicenseExpression], SpdxNoAssert, SpdxNone]
+    license_comments: Optional[str]
```

Comment:
Even though this is plural in the spec, I would suggest using singular here as it is not a list of strings.

Reply:
Renamed.
I thought about this when writing the property and concluded that one string would usually hold multiple comments. But I agree that it's better to make the variable name singular, especially since we do the same for other properties

Reply:
Sorry for not catching that earlier, but the same applies to packages and files

Reply:
renamed